### ISSUE TITLE
startup messages that should not be there (0.45.0)

### ISSUE THREAD

**Issue by sinteur** [RocketChat/Rocket.Chat#4922]
ufs: permissions are not defined for store "fileSystem"
ufs: store created at

[Thu Nov 10 2016 19:20:56 GMT+0100 (CET)] WARNING A script has tried registering a HTTP route while the HTTP server is disabled with --disabled-httpd.

**Comment by reetp** [comment]
I updated to 0.46.0 on CentOS 6 and get similar so I guess this is ongoing:

WARNING A script has tried registering a HTTP route while the HTTP server is disabled with --disabled-httpd

ufs: permissions are not defined for store "rocketchat_uploads"

I am running a Apache reverse SSL Proxy. File store is GridFS. 

By default I can upload but not download files.

I set Protect uploaded files to False and although I still get the error 'permissions not defined error' I can see/download files
